diff --git a/lib/bluetooth-client.c b/lib/bluetooth-client.c
index fd75bbf..20e6560 100644
--- a/lib/bluetooth-client.c
+++ b/lib/bluetooth-client.c
@@ -407,7 +407,7 @@ device_added (GDBusObjectManager   *manager,
 
 	device_resolve_type_and_icon (device, &type, &icon);
 
-	if (name == NULL && type == BLUETOOTH_TYPE_ANY)
+	if (name == NULL && type == 0)
 		return;
 
 	if (get_iter_from_path (priv->store, &parent, adapter_path) == FALSE)
