From 82d3e94e400be60d4f7ad6905d1a315521ab73ad Mon Sep 17 00:00:00 2001
From: HaeRyong Jeong <jeong89@gooroom.kr>
Date: Wed, 5 Feb 2020 16:17:18 +0900
Subject: [PATCH] pin code dialog cancel button fix

---
 lib/bluetooth-pairing-dialog.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/bluetooth-pairing-dialog.c b/lib/bluetooth-pairing-dialog.c
index 4565eeb..627d02a 100644
--- a/lib/bluetooth-pairing-dialog.c
+++ b/lib/bluetooth-pairing-dialog.c
@@ -230,6 +230,7 @@ response_cb (GtkWidget *button,
 		g_assert_not_reached ();
 
 	gtk_dialog_response (GTK_DIALOG (user_data), response);
+    gtk_widget_destroy (GTK_WIDGET (user_data));
 }
 
 static void
-- 
2.20.1

